chains/container-chains/runtime-templates/frontier/src/xcm_config.rs

```diff
@@ -229,6 +230,7 @@ impl xcm_executor::Config for XcmConfig {
     type HrmpChannelAcceptedHandler = ();
     type HrmpChannelClosingHandler = ();
     type XcmRecorder = ();
+    type XcmEventEmitter = (); // TODO: revisit this
```

Comment:
```suggestion
    type XcmEventEmitter = PolkadotXcm;
```

For other runtimes, either PolkadotXcm or XcmPallet

Using () means no events will be emitted, which I believe matches old behavior before this item was introduced:

https://github.com/paritytech/polkadot-sdk/pull/7234